_data/simple-icons.json

```
@@ -2075,6 +2075,11 @@
             "source": "https://beatstars.world/brand-guidelines",
             "guidelines": "https://beatstars.world/brand-guidelines"
         },
+        {
+            "title": "beecrowd",
+            "hex": "662C83",
```

```suggestion
            "hex": "521D69",
```

This is the accent color I'm seeing on their website.

---

Looks like you updated the color but it still doesn't appear correct, I believe it should be `#521D69` unless you see/find otherwise.

---

But I find #662483 as the main color, though I looked manually but If you are sure then I can set the color to #521D69

---

Ah sure actually I see that now. You are correct, `#662483` is the right hex.